buildtools/templates/index.tpl

```diff
@@ -17,6 +17,12 @@
     var options = {{{options}}};
     var signIn = new OktaSignIn(options);
 
+    signIn.on('authError', function (err) {
+      if (err.name=='AuthApiError') {
+        console.log(err);
```

Comment:
Is this just to make debugging easier right?

Reply:
Yeah, I guess so. I wanted to replicate GSD's changes as close as possible, to start with. We can probably remove this later. 👍